src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -788,10 +790,11 @@ class ApplicationManagerImpl
   void OnFindNewApplicationsRequest() OVERRIDE;
   void RemoveDevice(
       const connection_handler::DeviceHandle& device_handle) OVERRIDE;
-  bool OnServiceStartedCallback(
+  void OnServiceStartedCallback(
```

Comment:
`connection_handler().NotifyServiceStartedResult(...)` is called instead of returning a value.